pkg/plugins/manager/loader/angular/angularinspector/angularinspector_test.go

```diff
@@ -93,6 +93,8 @@ func TestDefaultStaticDetectorsInspector(t *testing.T) {
 		plugin *plugins.Plugin
 		exp    bool
 	}
+
+	//nolint:prealloc // just a test, and it'd require too much refactoring to preallocate
```

Comment:
Nit. Maybe skip this rule for tests if possible?

Reply:
I'm ambivalent - I prefer to write test code as if it were production code as much as possible, so when it's not too awkward in a test I think it's a good idea to preallocate.